examples/formik/js/formik-components/readme.md

```diff
@@ -0,0 +1,17 @@
+# Formik Components
+
+Example integrating `fonk` with `formik`.
+
+Using Field Formik component.
+
+[![See validate-field example](https://codesandbox.io/static/img/play-codesandbox.svg)](https://codesandbox.io/s/github/lemoncode/fonk/tree/master/examples/react-final-form/js/formik-component)
```

Comment:
Typo `formik-components`. The folder name is with `s`

Reply:
Good point and replace react-final-form with formik. ouch :), thanks